.github/workflows/test_on_push.yaml

```diff
@@ -12,15 +12,15 @@ jobs:
     runs-on: "ubuntu-latest"
```

Comment:
The latest is failing on some actions, Jobbergate went around it by pinning the os: https://github.com/omnivector-solutions/jobbergate/blob/1b2498040828f8e63945ac4140f76e02e338a4e6/.github/workflows/test_on_push.yaml#L12

Reply:
Not a problem for armasec because it uses python 3.8 as the minimum. That issue was because of supporting python 3.6.